src/test/kotlin/org/taktik/icure/testutils/TestDb.kt

```diff
@@ -0,0 +1,39 @@
+package org.taktik.icure.testutils
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.taktik.couchdb.id.UUIDGenerator
+import org.taktik.icure.asyncdao.impl.CouchDbDispatcher
+import org.taktik.icure.config.CouchDbConfig
+import org.taktik.icure.properties.CouchDbProperties
+
+object TestDb {
+	private const val PREFIX = "test-volatile"
+// 	private val databaseHost = System.getProperty("icure.test.couchdb.server.url")
```

Comment:
Better to use those than clear credentials (even if it's for a local DB)

Reply:
It was actually unused, it was from the first tests I made but then replaced the db with a mock.